resources/js/components/SignUp.vue

```diff
@@ -1,24 +1,24 @@
 <template>
     <div>
-        <nav class="navbar navbar-dark bg-primary justify-content-center">
+        <div class="npy-5 text-white bg-primary justify-content-center pt-5 pb-5">
             <h1 class="text-center">Badges</h1>
-        </nav>
+        </div>
         <div class="container">
             <div class="row justify-content-center">
                 <form @submit.prevent="signup" class="panel">
```

Comment:
Why does the form have a class of "panel"? It is not achieving anything, so please remove.